### DISCUSSION TITLE
Is it possible to add items to the queue in a script/automation?

### DISCUSSION THREAD

**Discussion by freekeys** [music-assistant/support#676] [Q&A] [answered]
Or alternatively, add albums to a playlist?

**Comment by marcelveldt** [comment, marked as answer]
Yes, you can use the "enqueue" options for that in the play_media service call. Those were added especially with Music Assistant in mind ;-)

So you can for example use "enqueue: add" to add items to the queue.
And items may be any media item, so even  playlists by itself

**Reply by freekeys** [reply]
That's amazing, thanks!
